bot.rb

```diff
@@ -337,6 +337,13 @@ def process_code_signing(issue)
       return body.join("\n\n")
     end
 
+    def last_responding_user(issue)
+      client.issue_comments(SLUG, issue.number)
```

Comment:
The return value of this function call isn't used or am I missing something?

Reply:
Yeah the GH API is a bit weird, you have to make the call so the API call is made. This returns the model objects, so to get the actual response object you have to make a separate call which I do in the next line:

```ruby
link_to_last_page = client.last_response.rels[:last]
```

Reply:
Oh, I see, thank you!